index.html

```diff
@@ -0,0 +1,166 @@
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous" />
+
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4" crossorigin="anonymous">
+        </script>
+</head>
+
+<body>
+    <header>
+        <div class="top-menu text-white">
+            <ul>
+                <li><i class="bi bi-twitter"></i></li>
+                <li><i class="fa fa-facebook"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </header>
+
+    <section>       
+        <nav>
+            <div class="navbar">
+                <div class="container nav-container">
+                    <input class="checkbox" type="checkbox" name="" id="" />
+                    <div class="hamburger-lines">
+                        <span class="line line1"></span>
+                        <span class="line line2"></span>
+                        <span class="line line3"></span>
+                        <div><img src="img/CS-logo.png" alt=""></div>
+                    </div>
+                    
+                    <a href="index.html"><img src="img/CS-logo.png" alt="Page Logo"></a>
+                    <ul class="nav-items">
+                        <li class="nav-item"><a href="about.html">About</a></li>
+                        <li class="nav-item"><a href="#">Program</a></li>
+                        <li class="nav-item"><a href="#">Join</a></li>
+                        <li class="nav-item"><a href="#">Sponsor</a></li>
+                        <li class="nav-item"><a href="#">News</a></li>
+                    </ul>
+                    <button class="nav-button">CS Campaign</button>
+                </div>
+            </div>
+        </nav>
+
+        <div class="header-content-container">
+            <div class="header-content">
+                <p class="content-title">"Hello! Sharing world"</p>
+                <h1>DIGITAL TRANSFORMATION<br>
+                    AFRICAN SUMMIT <br class="none">
+                </h1>
+                <div class="header-content-body">
+                    <p>The digital landscape is constantly evolving, and businesses and individuals must adapt to stay relevant.
+                       This web summit aims to explore the concept of digital transformation and how organizations can leverage technology to thrive in the future.
+                    </p>
+                </div>
+                <h3 class="summit-date">2023.09.11(WED) ~ 13(FRI)</h3>
+                <p class="summit-venue">@  Wekalet El Ghouri Arts Center Egypt</p>
+            </div>
+        </div>
+    </section>
```

Comment:
as suggested previously the text-align property of these texts is not centred, but justified. Kindly stick to the design by fixing that 😄

![image](https://github.com/Edgahkipkemoi/capstone/assets/60612329/2bef8ac0-9edc-483a-a37c-c50ce8a0ca57)